Library/Formula/watch.rb

```diff
@@ -12,15 +18,47 @@ class Watch < Formula
     sha1 "a7c559378bc74cd30d00f962e63d6ee5c705aea1" => :mountain_lion
   end
 
+  depends_on "gettext" => :build
```

Comment:
FYI: This isn't a build-time dep if you're building with Native Language Support. You remove it and runtime breaks.